field_internal_test.go

```diff
@@ -235,6 +236,15 @@ func (f *TestField) Reopen() error {
 	return nil
 }
 
+func (f *TestField) MustSetBit(row, col uint64, ts ...time.Time) {
```

Comment:
The name of this method could be more descriptive; it requires at least one timestamp to do anything, so perhaps `Time` should be part of the name. Also, it probably doesn't matter, but since it's only used in this test it doesn't need to be exported.

Reply:
I don't think exported or not matters for things in tests - nothing external can access them anyway. Agree about the name though - I'm going to change it so that the timestamp is actually optional.